### PR TITLE
Make it work with upcoming Nim 0.12.0

### PR DESCRIPTION
**Pull request by Araq**

### CODE REVIEW

File: jester.nim [hunk 1]
```diff
@@ -159,24 +159,24 @@ proc createReq(jes: Jester, path, body, ip: string, reqMeth: ReqMeth, headers,
...
+  if result.headers.getOrDefault("Content-Type").startswith("application/x-www-form-urlencoded"):
     try:
       parseUrlQuery(body, result.params)
     except:
       logging.warn("Could not parse URL query.")
-  elif result.headers["Content-Type"].startsWith("multipart/form-data"):
-    result.formData = parseMPFD(result.headers["Content-Type"], body)
-  if result.headers["SERVER_PORT"] != "":
-    result.port = result.headers["SERVER_PORT"].parseInt
+  elif (let ct = result.headers.getOrDefault("Content-Type"); ct.startsWith("multipart/form-data")):
```
Comment (dom96) [inline jester.nim]: Why introduce a variable here? and everywhere else?

Comment (Araq) [inline jester.nim]: Efficiency. 1 lookup instead of 2. Plus I find it easier to read.

Comment (dom96) [inline jester.nim]: Is
```nim
elif (let ct = result.headers.getOrDefault("Content-Type"); ct.startsWith("multipart/form-data")):
```

really more efficient than

```nim
elif result.headers.getOrDefault("Content-Type").startsWith("multipart/form-data"):
```

Two of those access ``Content-Type``, the ``let`` should really go above the if statement.

Comment (Araq) [inline jester.nim]: The 'ct' is accessed again in the 'then' section of the if statement. You can move the 'let' above the 'if' though if you dislike (;) so much. ;-)

Comment (Araq) [inline jester.nim]: Either way, please merge and adapt it to your liking. Right NOW Jester cannot be used with devel!

**Pull request merged**